etc/tempesta_fw.conf

```diff
@@ -1189,6 +1189,18 @@
 #   http_max_header_list_size 4096;
 #
 
+# TAG: http_allow_empty_body_content_type
+#
+# This directive controls whether HTTP requests are allowed to include
+# a Content-Type header even when the message body is empty.
+#
+# Syntax:
+#   http_allow_empty_body_content_type true|false
+#
+# Example:
+#   http_allow_empty_body_content_type true;
+#
```

Comment:
Could you please also describe this in wiki? Also with good story from https://github.com/tempesta-tech/tempesta/issues/2322 and https://github.com/tempesta-tech/tempesta/pull/2341#issue-2845726528 : make examples for the implementations, cite the RFC and how to we violate it (e.g. with `Content-Length: 0`).

Reply:
DONE: https://github.com/tempesta-tech/tempesta/wiki/HTTP-security#relaxed-content-type-and-content-length-rules